### ISSUE TITLE
A lot of spurious warnings after upgrading 1.7.0 -> 1.7.1

### ISSUE THREAD

**Issue by jorisvandenbossche** [sphinx-doc/sphinx#4685]
On the pandas doc build on travis, once it started using 1.7.1, there are a lot of warnings in the log output like (see eg https://travis-ci.org/pandas-dev/pandas/jobs/347059156, but there is a lot of other noise in the log output): 

```
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
```

that were not there before (with 1.7.0). 
It might be that those warnings are trying to warn for actual cases where there is a problem with this. But, the problem is they are not giving any context (in which file, which line is is raised?), so are somewhat useless.

**Comment by tk0miya** [comment]
Thank you for reporting. The warnings are emitted on creating autosummary tables. But it is not a problem of markups.
That's only bug of autosummary.

**Comment by jorisvandenbossche** [comment]
And thanks for fixing!

**Comment by tk0miya** [comment]
Note: I made following Dockerfile to build docs of pandas module.

```
FROM tk0miya/sphinx-html

ENV DISPLAY :99
RUN apt update
RUN apt install -y python3-tk xvfb libsnappy-dev pandoc
RUN git clone https://github.com/pandas-dev/pandas
WORKDIR /docs/pandas
RUN pip3 install cython nbconvert nbsphinx ipython matplotlib xarray xlrd openpyxl jupyter_client tables statsmodels pyarrow sqlalchemy fastparquet python-snappy feather-format lxml
RUN pip3 install -e .
WORKDIR /docs/pandas/doc
RUN rm -f /tmp/.X99-lock && \
    /usr/bin/Xvfb :99 & \
    while [ ! -f /tmp/.X99-lock ]; do sleep 1; done && \
    python3 make.py
```

**Comment by tk0miya** [comment]
Fixed by #4696.
Thank you for reporting.
